### PR TITLE
chore(errors): add len to returns the fields

### PR DESCRIPTION
**Pull request by mrbardia72**
add len to method Fields returns the fields that failed validation

### CODE REVIEW

File: foundation/validate/errors.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (fe FieldErrors) Error() string {
 
 // Fields returns the fields that failed validation
 func (fe FieldErrors) Fields() map[string]string {
-	m := make(map[string]string)
+	m := make(map[string]string,len(fa))
```
Comment (ardan-bkennedy) [inline foundation/validate/errors.go:39]: I don't think this was fmt'd. I would expect a space after the comma.

**Comment by ardan-bkennedy** [comment]
Are you already in the CONTRIBUTORS file?

**Comment by mrbardia72** [comment]
No, I had added it.

**Comment by ardan-bkennedy** [comment]
Not sure I understand your last response. Can you check that you are in the CONTRIBUTORS file please and if not, add yourself. Let me know that you are already.

**Comment by mrbardia72** [comment]
> Not sure I understand your last response. Can you check that you are in the CONTRIBUTORS file please and if not, add yourself. Let me know that you are already. 

Added to the CONTRIBUTORS file

**Pull request merged**